service-front/module/Application/src/View/Helper/RendererInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+// @inheritdoc
```

Comment:
remove this comment, and replace with autodocs. see https://www.jetbrains.com/help/phpstorm/phpdoc-comments.html if this helps.

Reply:
also see https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html

Reply:
Done